plugins/@grouparoo/demo/src/events.ts

```diff
@@ -34,7 +33,7 @@ export async function events() {
 
   const apiKey = await getApiKey();
   const categories = await getPurchaseCategories();
-  const purchases = await getPurchases();
+  const purchases = await getPurchases(250);
```

Comment:
Should this be a function of `--scale?`

Reply:
well this is base for `scale` and then that number can scale it up.